secant/Features/Settings/SettingsView.swift

```diff
@@ -41,6 +53,77 @@ struct SettingsView: View {
     }
 }
 
+extension SettingsView {
+    private func exportAndShareLogs() {
```

Comment:
I think this should be on the store instead of the View itself

Reply:
Reimplemented so the business logic is in the TCA while presentation layer is in the View.